main.go

```diff
@@ -440,7 +440,8 @@ func getUserFromContext(ctx context.Context) model.ADBUser {
 var verifier = func() *oidc.IDTokenVerifier {
 	provider, err := oidc.NewProvider(context.Background(), "https://accounts.google.com")
 	if err != nil {
-		log.Fatal(err)
+		log.Println("WARNING: failed to construct OIDC provider (no Internet connection?). Some features may not work: ", err)
```

Comment:
Small nit: Println automatically adds spaces between each operand, so you don't need a space after the colon (unless you were trying to print two spaces).

Reply:
Thanks for the tip, fixed it